src/security/Crypto.js

```diff
@@ -51,7 +51,7 @@ class Crypto {
      * @param {Object} keyPair - such as returned by Token.Crypto.generateKeys
      * @return {Object} signer object
      */
-    static createSignerFromKeypair(keyPair) {
+    static createSignerFromKeyPair(keyPair) {
```

Comment:
are these methods only used within the SDK? if not, then this could be breaking change so be careful

Reply:
yeah thought about it, its only used in the sdk afaik

Reply:
i guess we dont have to take that risk, even if its 1% chance